tests/test_analyze.py

```
@@ -0,0 +1,80 @@
+import pytest
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)